client/src/components/user-admin/UserManagement.js

```diff
@@ -42,69 +49,108 @@ const UserManagement = ({ users, setUserToEdit }) => {
             );
   }
   return (
-    <div className="container--usermanagement">
-      <div>
-        <h3>User Management</h3>
-        <input
+    <Box className="container--usermanagement" sx={{px: '1.8rem'}}>
+      <Box sx={{
```

Comment:
This box needs either `width: 1/1` or `width: "100%"` to fill the contents of it's parent.